test/tbb/test_parallel_invoke.cpp

```diff
@@ -29,6 +29,7 @@
 
 //! Testing parallel_invoke memory usage
 //! \brief \ref resource_usage \ref stress
+#if !EMSCRIPTEN
```

Comment:
Lets be aligned with other Test cases and move macro above comment.
And please add a comment on why all of these test cases are disabled for WASM.

Reply:
Please add comments why they are disabled on WASM.

Reply:
Done.